src/neo/IO/Json/JObject.cs

```diff
@@ -25,44 +40,94 @@ public class JObject
             }
         }
 
+        /// <summary>
+        /// Converts the current JSON object to a boolean value.
+        /// </summary>
+        /// <returns>The converted value.</returns>
         public virtual bool AsBoolean()
         {
             return true;
         }
 
+        /// <summary>
+        /// Converts the current JSON object to a floating point number.
+        /// </summary>
+        /// <returns>The converted value.</returns>
         public virtual double AsNumber()
         {
             return double.NaN;
         }
 
+        /// <summary>
+        /// Converts the current JSON object to a <see cref="string"/>.
+        /// </summary>
+        /// <returns>The converted value.</returns>
         public virtual string AsString()
         {
             return ToString();
         }
 
+        /// <summary>
+        /// Determines whether the JSON object contains a property with the specified name.
+        /// </summary>
+        /// <param name="key">The property name to locate in the JSON object.</param>
+        /// <returns><see langword="true"/> if the JSON object contains a property with the name; otherwise, <see langword="false"/>.</returns>
         public bool ContainsProperty(string key)
         {
             return Properties.ContainsKey(key);
         }
 
+        /// <summary>
+        /// Converts the current JSON object to a <see cref="JArray"/> object.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <exception cref="InvalidCastException">The JSON object is not a <see cref="JArray"/>.</exception>
         public virtual JArray GetArray() => throw new InvalidCastException();
 
+        /// <summary>
+        /// Converts the current JSON object to a boolean value.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <exception cref="InvalidCastException">The JSON object is not a <see cref="JBoolean"/>.</exception>
         public virtual bool GetBoolean() => throw new InvalidCastException();
 
+        /// <summary>
+        /// Converts the current JSON object to a 32-bit signed integer.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <exception cref="InvalidCastException">The JSON object is not a <see cref="JNumber"/>.</exception>
+        /// <exception cref="InvalidCastException">The JSON object cannot be converted to an integer.</exception>
```

Comment:
Duplicate `InvalidCastException` ?

Reply:
No duplicates. They are two different cases and both throw `InvalidCastException`.